src/database/entity/Appointment.ts

```diff
@@ -11,9 +11,11 @@ export class Appointment {
     @PrimaryGeneratedColumn()
     public id: number;
 
-    @Column({ type: "timestamp without time zone" })
-    public startDate: number;
+    // startDate is Unix Timestamp (seconds since 01.01.1970(UTC))
```

Comment:
a nie milisekundy ?

Reply:
O ile się nie mylę, to wcześnie też z sekund korzystałem, ale to kwestia dogadania, mogę zmienić tu i na froncie :)

Reply:
Jeszcze wygooglowałem czy się nie mylę, ale Unix Timestamp to sekundy. Milisekundy też nie są nam potrzebne w przypadku rejestracji wizyt